File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int
while True:
    N = get_int("Enter height: ")
    if N >= 1 and N <= 8:
        break


for i in range (N):
    for j in range (N):
        if (i+j) < N - 1:
            print(" ", end="")
        else:
            print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int N;
    do
    {
        N = get_int("Enter a number between 1 and 8\n");
    }
    while (N < 1 || N > 8);
    for (int i = 0; i < N; i++)
    {
        for (int j = 0; j < N; j++)
        {
            if (i + j < N - 1)
            {

                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}